sdk/storage/Azure.Storage.DataMovement/tests/GetTransfersTests.cs

```diff
@@ -216,5 +216,136 @@ public async Task GetTransfers_LocalCheckpointer()
             // Assert
             Assert.AreEqual(checkpointerTransfers, result.Select(d => d.Id).ToList());
         }
+
+        [Test]
+        public async Task GetResumableTransfers_LocalCheckpointer()
+        {
+            // Arrange
+            using DisposingLocalDirectory test = DisposingLocalDirectory.GetTestDirectory();
+            string parentRemotePath = "https://account.blob.core.windows.net/resume-test/";
+            string parentLocalPath1 = "/resume-test/";
+            string parentLocalPath2 = @"C:\Windows\Path\";
```

Comment:
Based on earlier comment with string parsing I recommend paths that both do and don't have terminating slashes. You can use Path.Combine() for when you add an extra segment to ensure path additions are always correct regardless of trailing slash state.

Reply:
Changing this parent path (or adding a new one) wouldn't actually change the test. Instead, what I did is made one of the single tests have slashes at the end of the file name and when generating filenames for the container, one file in the container will always end in a slash now.

Though again, a slash on the end of the file name should not matter for the logic to get the parent path (good to test anyway though)